app/app.py

```diff
@@ -45,28 +45,68 @@ def dashboard():
                 html.H1("🤩 Live Data 🤩"),
                 dcc.Graph(
                     figure=figure_service.get_number_of_repositories_with_standards_label_dashboard(),
-                    style={"width": "100%", "height": "500px", "display": "inline-block"},
+                    style={
+                        "width": "100%",
+                        "height": "500px",
+                        "display": "inline-block",
+                    },
                 ),
                 html.H1("🙈 Stub Data 🙈"),
                 dcc.Graph(
                     figure=figure_service.get_stubbed_number_of_repositories_with_standards_label_dashboard(),
-                    style={"width": "33%", "height": "500px", "display": "inline-block"},
+                    style={
+                        "width": "33%",
+                        "height": "500px",
+                        "display": "inline-block",
+                    },
                 ),
                 dcc.Graph(
                     figure=figure_service.get_stubbed_number_of_repositories_archived_by_automation(),
-                    style={"width": "33%", "height": "500px", "display": "inline-block"},
+                    style={
+                        "width": "33%",
+                        "height": "500px",
+                        "display": "inline-block",
+                    },
                 ),
                 dcc.Graph(
                     figure=figure_service.get_stubbed_sentry_transactions_used(),
-                    style={"width": "33%", "height": "500px", "display": "inline-block"},
+                    style={
+                        "width": "33%",
+                        "height": "500px",
+                        "display": "inline-block",
+                    },
                 ),
                 dcc.Graph(
                     figure=figure_service.get_support_stats(),
-                    style={"width": "100%", "height": "500px", "display": "inline-block"},
+                    style={
+                        "width": "100%",
+                        "height": "500px",
+                        "display": "inline-block",
+                    },
                 ),
                 dcc.Graph(
                     figure=figure_service.get_github_actions_quota_usage(),
-                    style={"width": "100%", "height": "500px", "display": "inline-block"},
+                    style={
+                        "width": "100%",
+                        "height": "500px",
+                        "display": "inline-block",
+                    },
+                ),
+                dcc.Graph(
```

Comment:
Happy to keep this here for now, but we should probably move it above the `Stub Data` section ♻️ 

Also might be better to create a new folder to hold production data i.e. data/production or something along those lines 😁